.github/ISSUE_TEMPLATE/new_repo.yml

```diff
@@ -0,0 +1,47 @@
+name: 🆕 New repo
+description: Request a repo
+title: "[New repo]: New Terraform module repo to automate <THIS>"
```

Comment:
should we omit `to automate <THIS>` ? This should be included repo description or more information fields. 
I can just picture people not updating THIS and its included in the git issue title

Reply:
I didn’t like `THIS` either. Are you recommending that we leave the title as New Terraform repo so all issues are generic or something else?